src/app/components/core/control-dispatcher/pipelines/control-pipelines.component.html

```diff
@@ -0,0 +1,93 @@
+<div class="container is-fluid">
+  <app-add-dispatcher-service></app-add-dispatcher-service>
+  <div class="card">
+    <header class="card-header">
+      <div id="north-tasks" class="card-header-title">
+        Control Pipelines
+      </div>
+      <a *ngIf="rolesService?.hasEditPermissions()" class="fix-pad button is-light" (click)="addControlPipeline()"
+        role="'admin'">
+        <p *ngIf="viewPort !== 'mobile'">Add &nbsp;</p>
+        <i class="fa fa-xs fa-plus" aria-hidden="true"></i>
+      </a>
+    </header>
+    <div class="card-content">
+      <div class="content">
+        <div *ngIf="showSpinner" class="app-loading">
+          &nbsp;
+          <svg class="spinner" viewBox="0 0 100 100">
+            <circle class="path" cx="50%" cy="50%" r="10" fill="none" stroke-width="2" stroke-miterlimit="10" />
+          </svg>
+        </div>
+        <ng-container *ngIf="!showSpinner">
+          <div class="has-text-centered" *ngIf="pipelines != undefined && pipelines.length === 0">
+            <small class="no-rec">No Record</small>
+          </div>
+          <ng-container *ngIf="pipelines!= undefined && pipelines.length > 0">
+            <table id="pipeline-list" class="table is-fullwidth scroll is-responsive is-hoverable pipeline-table"
+              cellpadding="0" cellspacing="0">
+              <thead>
+                <tr>
+                  <th class="align-content">Name</th>
+                  <th class="align-content">Status</th>
```

Comment:
We should have it Enabled checkbox as first column; or at-least make all such enabled to outlined